### DISCUSSION TITLE
What about the Servo Foundation?

### DISCUSSION THREAD

**Discussion by XX** [servo/servo#27710] [Ideas] [answered]
Is there a plan to create a foundation to support Servo development? I plan to use some crates (pathfinder, etc.), but I'm worried about the future of the Servo project. Maybe I could donate or contribute if the project needs help. Do you plan to organize an independent browser development?

**Comment by jdm** [comment]
What worries do you have about using those crates right now?

**Reply by XX** [reply]
Will their development stop due to the reduction of Servo's team in the Mozilla?

**Reply by jdm** [reply]
It think it's too soon to tell. I know that there were plans to adapt Pathfinder for use in Firefox, but I don't know the timeline on that work.

**Comment by SimonSapin** [comment, marked as answer]
The Servo Project is now hosted by the Linux Foundation:

* https://blog.servo.org/2020/11/17/servo-home/
* https://www.linuxfoundation.org/press-release/2020/11/open-source-web-engine-servo-to-be-hosted-at-linux-foundation/
